### DISCUSSION TITLE
[Docs][Checkbox]: outdated documentation for Checkbox component

### DISCUSSION THREAD

**Discussion by fluoxe7ine** [chakra-ui/zag#1107] [General]
# 🐛 Bug report

Checkbox is built from four parts: root, label, control and indicator. However, "indicator" part is not mentioned at all in the docs.

## 💥 Steps to reproduce

1. Go to https://zagjs.com/components/react/checkbox
2. Check that docs page doesn't consist any info about "indicator" part.

## 🧐 Expected behavior

Documentation for checkbox component consists information about "indicator" part, correct visualization image,  usage examples etc.

## 📝 Additional information

This issue is also related in [Ark UI documentation](https://ark-ui.com/docs/components/checkbox) too. Wrong anatomy visualization and no info about "indicator" part.

**Comment by segunadebayo** [comment]
Closing — documentation has been fully rebuilt for v1.0.
